fabric-item-group-api-v1/src/main/java/net/fabricmc/fabric/api/itemgroup/v1/IdentifiableItemGroup.java

```diff
@@ -14,21 +14,19 @@
  * limitations under the License.
  */
 
-package net.fabricmc.fabric.api.client.texture;
-
-import java.util.Set;
+package net.fabricmc.fabric.api.itemgroup.v1;
 
+import net.minecraft.item.ItemGroup;
 import net.minecraft.util.Identifier;
 
 /**
- * Implement this interface on a Sprite to declare additional dependencies
- * that should be processed prior to this sprite.
- *
- * <p>Best used in conjunction with {@link net.fabricmc.fabric.api.event.client.ClientSpriteRegistryCallback}.
+ * This interface is automatically implemented on all {@link ItemGroup} via Mixin and interface injection.
```

Comment:
```suggestion
 * This interface is automatically injected to {@link ItemGroup}.
```
This is how it's done in existing code (loot api), mixins are an impl detail